### ISSUE TITLE
Wrong boolean

### ISSUE THREAD

**Issue by R4YVEN** [Sn0wAlice/Alina#3]
you are setting "agePass" to true if the age is below 18?
maybe im blind but that doesnt look right. shouldnt it be the other way around?

if-block in question:
https://github.com/Sn0wAlice/Alina/blob/894e1ac8c3f3736dfa018a06e968d9aa2d49a18a/main.ts#L132

**Comment by Sn0wAlice** [comment]
yes, alina is a public tool to detect child abuse and contribute to authorithy...

yes xD good idea, but the script is quite dead for the moment... i will work on it later.  i let this issue open to remember. 
Thanks you
